gitlab.go

```diff
@@ -123,6 +123,18 @@ func (t *ISOTime) UnmarshalJSON(data []byte) error {
 	return err
 }
 
+// EncodeValues implements the query.Encoder interface
+func (t *ISOTime) EncodeValues(key string, v *url.Values) error {
+	if t == nil {
+		return nil
+	}
+	if (time.Time(*t)).IsZero() {
+		return nil
+	}
```

Comment:
I would place these two checks on one line:

```
	if t == nil || (time.Time(*t)).IsZero() {
		return nil
	}
```

Reply:
> I would place these two checks on one line:
> 
> ```
> 	if t == nil || (time.Time(*t)).IsZero() {
> 		return nil
> 	}
> ```

Should be resolved by 02c5e68630d5af6e610c03bded6741997230578e.